spec/models/guide_spec.rb

```diff
@@ -129,20 +129,29 @@
 
   describe "#with_published_editions" do
     it "only returns published editions" do
-      create(:guide, slug: "/service-manual/topic-name/1")
-      guide_with_published_editions = create(:published_guide, slug: "/service-manual/topic-name/2")
-      expect(Guide.with_published_editions.to_a).to eq [guide_with_published_editions]
+      guide_community = create(:guide_community, :with_published_edition)
+      create(:guide, slug: "/service-manual/topic-name/1", edition: {
+        content_owner_id: guide_community.id
+      })
+      published_guide = create(:guide, :with_published_edition,
+        slug: "/service-manual/topic-name/2",
+        edition: {
+          content_owner_id: guide_community.id
+        }
+      )
+
+      expect(Guide.with_published_editions.to_a).to eq [guide_community, published_guide]
```

Comment:
(you probably know this but order isn't guaranteed in postgres without an `ORDER BY`)